### DISCUSSION TITLE
Products missing from stores, no warning

### DISCUSSION THREAD

**Discussion by ThrowAwayArr** [jef/streetmerchant#1275] [Help]
## Expected Behavior

SHOW_ONLY_SERIES="rx6900xt,ryzen5900"

=> Seeing requests and listings of the two series

PAGE_SLEEP_MIN="300000"

=> Expecting a look up every 5 minutes

## Current Behavior

SHOW_ONLY_SERIES="rx6900xt,ryzen5900"

=> There are no lookups for rx6900xt whatsoever. Also tried rx6900 / 6900xt respectively to no avail. There are no log prints whatsoever, nor any errors. The 5900 lookups are happening just fine. rx6800xt seems to work fine though.

PAGE_SLEEP_MIN="300000"

=> The sleep is applied BEFORE the first cycle of lookups. I'd expect the first cycle to begin immediately and the sleep only kick in between cycles.

## Steps to Reproduce

Clone repo, create .env and add content below

## Environment

**OS: Linux**
**.env file:**

```
SHOW_ONLY_SERIES="rx6900xt,ryzen5900"
PAGE_BACKOFF_MIN="900000"
PAGE_BACKOFF_MAX=""
PAGE_SLEEP_MIN="300000"
PAGE_SLEEP_MAX="600000"
STORES="amd-de,alternate,amazon-de,amazon-uk,amazon-nl,amazon-fr,amazon-es,arlt,asus-de,caseking,computeruniverse,cyberport,mediamarkt,proshop-de,saturn,notebooksbilliger,euronics-de,mindfactory"
PLAY_SOUND="notification.mp3"
MAX_PRICE_SERIES_RYZEN5900="600"
MAX_PRICE_SERIES_RX6900XT="1200"
```

All other values are empty/default.

## Logs

VIsualization: https://imgur.com/a/uvjBFwQ

**Comment by jef** [comment]
Some of those stores may not have those models. They'll need to get added to be searched properly.

**Comment by ThrowAwayArr** [comment]
> Some of those stores may not have those models. They'll need to get added to be searched properly.

I see, so if their homepage searches yield no result of any kind it will not show up as having even made a request, instead of, say, 'Not Found/Listed'. Perhaps a log of any kind (like not found) would prevent confusion.

Either way, I'll then be a bit patient, for things have been a big mess in Europe.

The PAGE_SLEEP_MIN 'issues' is minor but might be annoying as it delays the first cycle notably, insofar one doesn't want to spam the shops with look ups.

Feel free to close this if you don't see a need to take any action(s).

Thanks for the work, it's a rather charming experience overall so far ;)

**Comment by burkj2003** [comment]
I'm also having this issue.  Even with the AMD direct buy 'store'.  Also, there are no AMD video card brands such as Sapphire or XFX (and others) listed as an available brand.

**Comment by jef** [comment]
> I'm also having this issue. Even with the AMD direct buy 'store'. Also, there are no AMD video card brands such as Sapphire or XFX (and others) listed as an available brand.

It could be an available brand, but it's may not be available in that store.

**Comment by burkj2003** [comment]
I found the 'issue' if anyone is interested.  The 6900XT's have not been added to the code yet.  So, if  you find a store that has the card you want then you need to do what Jef says above.  The link needs to be added to the code for that store.  Let's say you found the 6900XT links in Newegg (which are there as of this writing), then you need to edit the newegg.ts file to the list of available cards/links.  You'll see the various examples that have been entered.  I copied a 6800xt set of code, changed everything to 6900xt and then updated the link for the correct link.  Then the bot starts looking for a link that exists.  Repeat for any card/store you need.
